src/utils/Terminal.php

```diff
@@ -111,6 +121,16 @@ protected static function getFallbackEscapeCodes() : void{
 		self::$COLOR_YELLOW = $color(227);
 		self::$COLOR_WHITE = $color(231);
 		self::$COLOR_MINECOIN_GOLD = $color(184);
+		self::$COLOR_MATERIAL_QUARTZ = $color(188);
+        self::$COLOR_MATERIAL_IRON = $color(188);
```

Comment:
are these supposed to be the same?

Reply:
Essentially the same color

![image](https://github.com/pmmp/PocketMine-MP/assets/40174895/ff1cee2b-1ac8-4d6c-8911-5a5cc74cd901)

Reply:
According to https://www.ditig.com/256-colors-cheat-sheet and https://minecraft.fandom.com/wiki/Formatting_codes both of the colors can be distinctly different. From what I can see the closet for quartz is between 252-254 and iron is near perfect to 252

Reply:
Quartz looks slightly lighter to me.

Reply:
> According to https://www.ditig.com/256-colors-cheat-sheet and https://minecraft.fandom.com/wiki/Formatting_codes both of the colors can be distinctly different. From what I can see the closet for quartz is between 252-254 and iron is near perfect to 252

![image](https://github.com/pmmp/PocketMine-MP/assets/40174895/59b293bc-4351-41e4-80b1-616b967101cd)

I used to use a converter, but I've just redone it myself thanks to the site you sent, which is much more complete than my old converter.